### PR TITLE
Introduce 3 issue templates and links

### LINKED ISSUE #1096: Add Issue Templates for Kpt Repository
Introduce a set of issue templates to make opening issues clearer and more friendly when interacting with the kpt repository.

See: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository for references of the template creation process.

### PR DESCRIPTION
**Pull request by runewake2**
Fixes #1096

Intended to streamline the process of opening new kpt issues.

Introduces three issue templates for the kpt repository for:

* Feature Requests
* Bug Reports
* Kpt Questions

Also introduces two links for new issues that link to:

* Kpt channel in the kubernetes.slack.com slack server
* The Kpt Guides

These links are presented as part of the "New Issue" flow.

Comments on phrasing and styling appreciated.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Kpt Slack Channel
+    url: https://kubernetes.slack.com/app_redirect?channel=kpt
+    about: Discuss Kpt Topics with the Kpt Team on Slack
+  - name: Kpt Guides
```
Comment (prachirp) [inline .github/ISSUE_TEMPLATE/config.yaml:6]: We also have an email group at https://googlecontainertools.github.io/kpt/contact/

**Review by prachirp — CHANGES_REQUESTED** [review]
Had a few comments

**Comment by mikebz** [comment]
@runewake2 is this ready for another review?

**Comment by runewake2** [comment]
Yes, my recent changes should address the review comments.

**Review by prachirp — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+### Expected behavior
+
+### Actual behavior
+
+### Information
+
+<!-- Please include any additional information related to your issue -->
+<!-- EX: Operating System -->
```
[the remark this answers is not in the document]
Comment (frankfarzan) [reply .github/ISSUE_TEMPLATE/bug.md:17]: Should help the user by showing exact command to get this information. e.g. 'kpt version'

In general, we should make it extremely easy for users to give us context needed.

**Review by frankfarzan — CHANGES_REQUESTED** [review]

**Comment by runewake2** [comment]
@frankfarzan PTAL

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+assignees: ""
+---
+
+### Expected behavior
+
+### Actual behavior
+
+### Information
+
+<!-- Please include any additional information related to your issue -->
```
Comment (frankfarzan) [inline .github/ISSUE_TEMPLATE/bug.md:16]: is this comment rendered and viewable by the user?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<!-- EX: Operating System -->
+<!--     Kpt Version (you may find this by running `kpt version` in your shell) -->
```
Comment (frankfarzan) [inline .github/ISSUE_TEMPLATE/bug.md:18]: IMO, we just need two pieces of info:

1. Output of `kpt version`. This should include any host info we need for debugging like platform and arch. If something is missing, it's easy to expand it.
2. Minimum reproducible package

File: .github/ISSUE_TEMPLATE/bug.md (hunk 3 above)
Comment (runewake2) [reply .github/ISSUE_TEMPLATE/bug.md:16]: Yes. Everything on this yaml page is rendered into the Issue's details. Here's the issue experience in kustomize which has similar templates: https://github.com/kubernetes-sigs/kustomize/issues/new/choose

If the comments are left in the issue they should not be rendered in the filed issues.

File: .github/ISSUE_TEMPLATE/bug.md (hunk 4 above)
Comment (mikebz) [reply .github/ISSUE_TEMPLATE/bug.md:18]: @runewake2 can you respond here and let's figure out how to bring this to closure 😄

Comment (mikebz) [reply .github/ISSUE_TEMPLATE/bug.md:18]: @runewake2 ping... lingering work...

Comment (runewake2) [reply .github/ISSUE_TEMPLATE/bug.md:18]: Should have time to loop back by the end of the week. I've been prioritizing other work. Thanks for the ping.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**